src/selectors/fridge.js

```diff
@@ -39,6 +29,41 @@ export const selectFridgeTemperatureLogs = createSelector([selectSelectedFridge]
   return temperatureLogs.sorted?.('timestamp') ?? [];
 });
 
+export const selectLeastRecentTemperatureLogDate = createSelector(
```

Comment:
bunch of these selectors were made already - going to do a new issue and change them a bit to try and utilize realm queries more than memoizing selectors